etest/ebuild.py

```diff
@@ -18,46 +19,46 @@
 class Ebuild(object):
     """Ebuild."""
 
-    def __init__(self, path, overlay):
+    def __init__(self, path: str, overlay: Any) -> None:
```

Comment:
Shouldnt `overlay` be of type `etest.overlay.Overlay`?

Reply:
Good catch, but due to the cyclic dependency we can't accomplish that in this pull request.